tst/sdp_config.h

```diff
@@ -0,0 +1,25 @@
+//
+// Session Description Protocol Configuration
+//
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_SDP_SDP_CONFIG__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_SDP_SDP_CONFIG__
+
+#include <inttypes.h>
+
+/**
+ * @brief SDP print format for uint64_t.
+ */
+#define SDP_PRINT_FMT_UINT64 PRIu64
```

Comment:
Its a requirement to pass the build by the newly added SDP component. We have this dummy file in configs too. Creating one exclusively for tst directory to separate out the source and tst components.

Reply:
Why do we need a specific one for tst directory?  Since the existing one is already at the configs/ directory and not in source, I think it would be much cleaner (and avoid code duplication) to simply reference the same file in the cmakelists for both the SDK and the test project (use target_include_directories).  I understand separating out configs in which the tests will override something but in this case we actually don't want the tests to override this as this is supposed to work for all platforms.